components/header/header.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import style from './header.module.scss';
+
+export default function Header() {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  return (
+    <div className={style.container}>
+      <div className={style.header}>
+        <a href="#" className={style.logo}>
+          <Image
+            alt="KedroLogo"
+            src="/images/kedro-logo.svg"
+            width={30}
+            height={30}
+          />
+          <h4 className={style.logoText}>Kedro</h4>
+        </a>
+        {/*TO DO - Update All Links*/}
```

Comment:
This can be deleted. ( P.S are the links updated?)

Reply:
get started is to be updated once the section is ready. so i will leave it for now